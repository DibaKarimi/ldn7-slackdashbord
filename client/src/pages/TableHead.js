import React from "react";
import "./Home.css";

const TableHead = (props) => {

  return (
    <tr>
      <th>
        Channel
      </th>
      <th>
        Description
      </th>
      <th>
        No. of users
      </th>
      <th>
        Users
      </th>
    </tr>
  );
};

export default TableHead;
