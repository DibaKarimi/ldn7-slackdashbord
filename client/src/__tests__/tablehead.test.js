import React from "react";
import TableHead from "../components/TableHead";

import { shallow } from "enzyme";

describe("TableHead", () => {
	it("should render table head correctly", () => {
		const component = shallow(<TableHead />);

		expect(component).toMatchSnapshot();
	});
});
