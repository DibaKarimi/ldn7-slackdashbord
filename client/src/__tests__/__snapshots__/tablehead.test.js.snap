// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead should render table head correctly 1`] = `
<tr
  className="p-5"
>
  <th>
    <img
      alt="Slack logo"
      className="slack_logo"
      src={null}
    />
    Channel name
  </th>
  <th>
    <img
      alt="Users icon"
      className="users_icon"
      src={null}
    />
    No. of users
  </th>
</tr>
`;
